google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/resources/ResourceEmptyModelItem.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.resources;
```

Comment:
Could you please elaborate a bit on what you did here, I'm a bit confused. This class for example is still in the `resources` package. Is that because its shared by other things?

Reply:
I'm not sure what the intent was. I assumed it was supposed to be general. Is this supposed to be specific to the repo integration?

Reply:
Actually I think this was also used by the old project selector. Although I'm not immediately seeing that..

Reply:
`ResourceErrorModelItem` is used by the project selector via `GoogleUserModelItem`. 

It could be that `ResourceEmptyModelItem` just wasn't renamed after a refactor.

Reply:
Some for `ResourceLoadingModelItem` -> its used by the project selector.

Reply:
I'm ok if you want to leave it in the `resource` package for now then and we take care of it separately.

Reply:
BTW, I was thinking about removing all `*ModelItem*` and old ProjectSelector classes once old project selector is completely replaced.